### PR TITLE
chore: generate props automatically for Button

### PR DESCRIPTION
**Pull request by denieler**
### Description

Replace manual props docs and use manually generated docs instead.

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

### CODE REVIEW

File: packages/picasso/src/Button/story/index.jsx [hunk 1]
```diff
@@ -23,80 +23,6 @@ page
...
-        description: 'Take the full width of a container',
-        defaultValue: 'false'
-      },
-      hovered: {
-        name: 'hovered',
-        type: 'boolean',
-        description: 'Set hovered style for the button',
-        defaultValue: 'false'
-      },
-      circular: {
```
Comment (denieler) [inline packages/picasso/src/Button/story/index.jsx:94]: here was a prop which doesn't exist anymore in the Button component 🤷‍♂️

File: packages/picasso/src/Button/Button.tsx [hunk 2]
```diff
@@ -49,13 +49,11 @@ export interface Props
     ButtonOrAnchorProps {
   /** Show button in the active state (left mouse button down) */
   active?: boolean
-  /** The component used for the root node. Either a string to use a DOM element or a component. */
   as?: ElementType
   /** Disables button */
   disabled?: boolean
   /** Content of Button component */
   children: ReactNode
-  // TODO: should it be exposed?
```
Comment (denieler) [inline packages/picasso/src/Button/Button.tsx:58]: it was not exposed 😃

**Review by TomasSlama — APPROVED** [review]
Nice, automation is always better. It is awesome that you recently fixed that we see exact values in the table, not name of the type

**Review by dmaklygin — APPROVED** [review]

**Review by OndrejTuma — APPROVED** [review]

File: packages/picasso/src/Button/Button.tsx (hunk 2 above)
Comment (LashaJini) [reply packages/picasso/src/Button/Button.tsx:58]: But `focused` is still not exposed :thinking:

Comment (denieler) [reply packages/picasso/src/Button/Button.tsx:58]: @LashaJini yes, so it was not exposed and kept to be not exposed 🤷 I've just removed this TODO label

**Pull request merged**